css/properties/text-underline-offset.json

```
@@ -4,9 +4,11 @@
       "text-underline-offset": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-underline-offset",
+          "spec_url": "https://www.w3.org/TR/css-text-decor-4/#underline-offset",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": false,
+              "notes": "Currently in development as indicated on Chrome Platform Status: <a href='https://chromestatus.com/feature/5747636764147712'>text-decoration-thickness, text-underline-offset and from-font keyword for text-underline-position</a>."
```

```suggestion
              "notes": "Currently in development, as indicated on Chrome Platform Status: <a href='https://chromestatus.com/feature/5747636764147712'>text-decoration-thickness, text-underline-offset and from-font keyword for text-underline-position</a>."
```

---

I don’t think we usually add notes like this for features that are in development but which haven’t shipped in any release yet. I think we’ll need @ddbeck to make a call on this.

---

We don't seek out information on features that aren't yet shipping, but we do have some incidental bug links etc. I'm OK with having one here, since other browsers are shipping, but we need to make sure it can't go stale easily. Anything that says "currently" is _very_ prone to going out-of-date (development priorities change all the time), so we should go with something super minimal here, such as:

```suggestion
              "notes": "See Chrome Platform Status entry for <a href='https://chromestatus.com/feature/5747636764147712'>text-decoration-thickness, text-underline-offset and from-font keyword for text-underline-position</a>."
```